packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -329,6 +342,72 @@ class ToolbarOptions {
   final bool selectAll;
 }
 
+/// Content insertion configuration for [EditableText].
```

Comment:
Nit: "Configures the ability to insert media content through the soft keyboard."